test/provision/dns.sh

```diff
@@ -3,8 +3,8 @@
 # This script update the dns servers to use google ones.
 set -e
 
-sudo systemctl disable systemd-resolved.service
-sudo service systemd-resolved stop
+sudo systemctl disable systemd-resolved.service || true
```

Comment:
just remove `set -e` instead ?

Reply:
@ldelossa could you please elaborate?

Reply:
Hmm maybe I'm mistaken, but the '|| true' is there just to handle the case whre the `systemctl` commands fail, but if you just remove `set -e` the script will continue to run, despite the possible failures?

Reply:
Gotcha, tnx! I think it makes sense.

Reply:
I'm fine either case, I'll keep the `|| true` given the linter comment from Joe.